doc/techref/projections.md

```diff
@@ -16,34 +16,34 @@ The table below shows the projection codes for the 31 GMT map projections:
+| **F**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | {doc}`/projections/azim/azim_gnomonic` |
+| **G**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | {doc}`/projections/azim/azim_orthographic` |
+| **G**{{ lon0 }}/{{ lat0 }}/*width*[**+a***azimuth*][**+t***tilt*][**+v***vwidth*/*vheight*][**+w***twist*][**+z***altitude*] | {doc}`/projections/azim/azim_general_perspective` |
+| **H**[{{ lon0 }}/]*width*                                   | {doc}`/projections/misc/misc_hammer` |
+| **I**[{{ lon0 }}/]*width*                                   | {doc}`/projections/misc/misc_sinusoidal` |
+| **J**[{{ lon0 }}/]*width*                                   | {doc}`/projections/cyl/cyl_miller` |
+| **Kf**[{{ lon0 }}/]*width*                                  | {doc}`/projections/misc/misc_eckertIV` |
+| **Ks**[{{ lon0 }}/]*width*                                  | {doc}`/projections/misc/misc_eckertVI` |
+| **L**{{ lon0 }}/{{ lat0 }}/{{ lat1 }}/{{ lat2 }}/*width*    | {doc}`/projections/conic/conic_lambert` |
+| **M**[{{ lon0 }}/[{{ lat0 }}/]]*width*                      | {doc}`/projections/cyl/cyl_mercator` |
+| **N**[{{ lon0 }}/]*width*                                   | {doc}`/projections/misc/misc_robinson` |
+| **Oa**{{ lon0 }}/{{ lat0 }}/*azimuth*/*width*[**+v**]       | {doc}`/projections/cyl/cyl_oblique_mercator_1` |
+| **Ob**{{ lon0 }}/{{ lat0 }}/{{ lon1 }}/{{ lat1 }}/*width*[**+v**] | {doc}`/projections/cyl/cyl_oblique_mercator_2` |
+| **Oc**{{ lon0 }}/{{ lat0 }}/{{ lonp }}/{{ latp }}/*width*[**+v**] | {doc}`/projections/cyl/cyl_oblique_mercator_3` |
+| **P***width*[**+a**][**+f**[**e**\|**p**\|*radius*]][**+r***offset*][**+t***origin*][**+z**[**p**\|*radius*]] | {doc}`Polar </projections/nongeo/polar` [azimuthal] ({math}`\theta, r`) (or cylindrical) |
+| **Poly**/[{{ lon0 }}/[{{ lat0 }}/]]*width*                  | {doc}`/projections/conic/polyconic` |
+| **Q**[{{ lon0 }}/[{{ lat0 }}/]]*width*                      | {doc}`/projections/cyl/cyl_equidistant` |
+| **R**[{{ lon0 }}/]*width*                                   | {doc}`/projections/misc/misc_winkel_tripel` |
+| **S**{{ lon0 }}/{{ lat0 }}[/*horizon*]/*width*              | {doc}`/projections/azim/azim_general_stereographic` |
+| **T**{{ lon0 }}[/{{ lat0 }}]/*width*                        | {doc}`/projections/cyl/cyl_transverse_mercator` |
+| **U***zone*/*width*                                         | {doc}`/projections/cyl/cyl_universal_transverse_mercator` |
+| **V**[{{ lon0 }}/]*width*                                   | {doc}`/projections/misc/misc_van_der_grinten` |
+| **W**[{{ lon0 }}/]*width*                                   | {doc}`/projections/misc/misc_mollweide` |
+| **X***width*[**l**\|**p***exp*\|**T**\|**t**][/*height*[**l**\|**p***exp*\|**T**\|**t**]][**d**] | {doc}`/projections/nongeo/cartesian_linear`, {doc}`/projections/nongeo/cartesian_logarithmic`, {doc}`/projections/nongeo/cartesian_power`, and time |
```

Comment:
Now the "projection name" for "-JX" is "Cartesian linear, Cartesian logarithmic, Cartesian power, and time". It's a little too long. 

I think for `-JX`, we should use the original version for simplicity.

Reply:
The titles are shorten in commit https://github.com/GenericMappingTools/pygmt/pull/3407/commits/6d9fb0525418e666fe3c845a0aa30574359dfb70.